src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -6621,6 +6621,7 @@ void MeasureNumberStateHandler::updateForMeasure(const Measure* const m)
     }
 
     // update measure numbers and cache result
+    _measureNo += m->noOffset();
```

Comment:
This seems to cause https://musescore.org/en/node/343581, a pretty bad issue with screwed up actual measure durations